src/components/DonarDashboard/MyProjects.tsx

```diff
@@ -437,14 +437,12 @@ const MyProjects = () => {
               </div>
               <div className='flex gap-1'>
                 <span className='font-medium text-[#1D1E1F]'>
-                  {formatAmount(claimableFees + claimedFees)} POL
+                  {formatAmount(claimedFees)} POL
                 </span>
                 <span className='font-medium text-[#82899A]'>
                   ~ ${' '}
                   {formatAmount(
-                    Math.round(
-                      (claimableFees + claimedFees) * Number(POLPrice) * 100,
-                    ) / 100,
+                    Math.round(claimedFees * Number(POLPrice) * 100) / 100,
```

Comment:
Instead of that let's use lodash `round`, we have used it in other places
[tempprogress = round((totalDonationAmountInRound / capAmount) * 100, 2); // Round to 2 decimal places](https://github.com/GeneralMagicio/q-acc/blob/265a8ad6a24840dbd181593cb321c6740be6bb01/src/components/DonatePage/DonatePageBody.tsx#L140)

Reply:
I believe we must replace rounding everywhere in the code to that, makes life easier